test/runner/lib/thread.py

```diff
@@ -33,7 +33,7 @@ def run(self):
         # noinspection PyBroadException
         try:
             self._result.put((self.action(), None))
-        except:  # pylint: disable=locally-disabled, bare-except
+        except BaseException:
```

Comment:
Keep the bare `except` here.

We want to catch all exceptions, even those that don't derive from `BaseException` under Python 2.x.

Of course there shouldn't be any of those, but just in case.

Reply:
Good point!  I forgot that Python2 still has old-style-class exceptions that aren't rooted at BaseException.